cpp/src/cider/function/CMakeLists.txt

```diff
@@ -99,6 +99,10 @@ add_custom_target(
   DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/RuntimeFunctions.bc
           ${CMAKE_CURRENT_BINARY_DIR}/ExtensionFunctions.ast)
 
+add_library(cider_runtime_function SHARED ${runtime_function_source_files})
+add_dependencies(cider_runtime_function QueryEngineFunctionsTargets)
+target_link_libraries(cider_runtime_function re2 substrait)
```

Comment:
why does cider_runtime_function depend on substrait?

Reply:
Not sure, need to clean up later.

Reply:
maybe we can remove it. cc @chaojun-zhang